app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb

```diff
@@ -131,6 +131,45 @@ def destroy_hosts
       respond_for_async :resource => task
     end
 
+    api :PUT, "/hosts/bulk/subscriptions/remove_subscriptions", N_("Remove subscriptions from one or more hosts")
+    param_group :bulk_params
+    param :subscriptions, Array, :desc => N_("Array of subscriptions to remove") do
+      param :id, String, :desc => N_("Subscription Pool id"), :required => true
+      param :quantity, Integer, :desc => N_("If specified, remove the first instance of a subscription with matching id and quantity"), :required => false
```

Comment:
> "If specified, remove the first instance of a subscription with matching id and quantity"

It could just be me, but this being on the quantity parameter is a bit confusing to me. I would expect it to say something like "quantity of specified subscription to remove" or something similar. Also since its not required, does it default to 1?